tests/test_tokenization_fsmt.py

```diff
@@ -86,6 +90,15 @@ def tokenizer_ru_en(self):
     def tokenizer_en_ru(self):
         return FSMTTokenizer.from_pretrained("facebook/wmt19-en-ru")
 
+    def test_online_tokenizer_config(self):
```

Comment:
-> test_fsmt_tiny2_params

Reply:
the test name mnemonics here are about testing the online config files, it doesn't matter what the model is - does it make sense?

Reply:
somewhat. Spending no more energy here :)